docs/development.md

```diff
@@ -0,0 +1,121 @@
+## Development
+> Guide to local development of this plugin
```

Comment:
It's not obvious to me why this is quoted. What do you think about this small formatting change?
```suggestion

Guide to local development of this plugin
```

Reply:
It's meant to be a subtitle. It's a pattern commonly followed in README.md files on GitHub. Though not consistently used.

I'll remove.

Reply:
e.g. https://github.com/MichaelCurrin/jekyll-blog-demo#jekyll-blog-demo---